### PR TITLE
Only package necessary files

### PR DESCRIPTION
**Pull request by Shinigami92**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,6 +46,9 @@
     "url": "https://github.com/pioardi/poolifier/issues"
   },
   "homepage": "https://github.com/pioardi/poolifier#readme",
+  "files": [
+    "lib"
```
Comment (pioardi) [inline package.json:50]: are we sure that this is going to work ? I mean should be lib/* or something like ?

**Comment by pioardi** [comment]
By the way should we add an npm run build to this workflow ? https://github.com/pioardi/poolifier/blob/master/.github/workflows/npmpublish.yml

File: package.json (hunk 1 above)
Comment (Shinigami92) [reply package.json:50]: Yes, you can check it via `npm publish --dry-run`

**Comment by Shinigami92** [comment]
> By the way should we add an npm run build to this workflow ? [https://github.com/pioardi/poolifier/blob/master/.github/workflows/npmpublish.yml](https://github.com/pioardi/poolifier/blob/master/.github/workflows/npmpublish.yml?rgh-link-date=2021-02-10T16%3A09%3A51Z)

YES 😬!!!

**Comment by Shinigami92** [comment]
> YES !!!

@pioardi Oh nevermind, I think it's working because the `test` script already has `npm run build` in it

**Review by jerome-benoit — APPROVED** [review]

**Pull request merged**